packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -126,7 +129,6 @@ const GitLabScopingSearchResults = (props: Props) => {
     query
   )
 
-  const lastItem = useLoadNextOnScrollBottom(paginationRes, {}, 20)
```

Comment:
Scrolling to the bottom of the page when the `projects` query contains a `projectsIds` causes an infinite loop. I spent time digging into this, but as we're refactoring this query in https://github.com/ParabolInc/parabol/issues/6325 and as the user can still find the desired issue using the search & filter menu, I think it's safe to remove for now.